icu.__language__.js

```diff
@@ -1 +1 @@
-icu.en.js
```

Comment:
Hmm, I don't think this change should occur, but I don't know what caused it. I'll probably revert this file and worry about what changed it another time.